### DISCUSSION TITLE
Set up click tracking?

### DISCUSSION THREAD

**Discussion by Praashanna** [postalserver/postal#3001] [Help with using Postal]
Even though I set up the CNAME for click.domain.tld pointing to postal.domain.tld, it shows "CNAME not configured correctly." What else shall I do to completely set up the click/open tracking?

Also, I saw that there are metrics in the route /health and /metrics? But I cannot access it. How do I see better metrics?

**Comment by willpower232** [comment]
If you edit the click tracking domain, it should tell you which domain it is looking for, usually something starting with `tracking.`

**Reply by Praashanna** [reply]
It worked and shows a green tick now. Still, the tracking isn't working even when I open the email!

Also,

> Also, I saw that there are metrics in the route /health and /metrics? But I cannot access it. How do I see better metrics?

Any way?

**Reply by Praashanna** [reply]
Also, I need to serve another web server on tracking domain right?

**Reply by willpower232** [reply]
If you visit the tracking URL directly then I think you should see the word Hello to confirm it is working.

You need to use caddy (or whatever you are using to proxy the web UI) to set up each tracking domain yes.

The metrics are for use with Prometheus but I don't know much about it.

**Comment by Praashanna** [comment]
finally got it, thank you...
